Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewController.swift

```diff
@@ -159,6 +159,7 @@ final class PledgePaymentMethodsViewController: UIViewController {
 
         assert(
           topViewController is PledgeViewController ||
+            topViewController is NoShippingPledgeViewController ||
             topViewController is PostCampaignCheckoutViewController,
           "PledgePaymentMethodsViewController is only intended to be presented as part of a pledge flow."
```

Comment:
We need to add this type-check here so that the Stripe payment sheet is presented on the `NoShippingPledgeViewController`